db/bigtable_eth1.go

```diff
@@ -1142,19 +1183,51 @@ func (bigtable *Bigtable) TransformItx(blk *types.Eth1Block, cache *freecache.Ca
 	bulkData = &types.BulkMutations{}
 	bulkMetadataUpdates = &types.BulkMutations{}
 
+	ContractUpdateWrites := &types.BulkMutations{
+		Keys: make([]string, 0),
+		Muts: make([]*gcp_bigtable.Mutation, 0),
+	}
+
 	for i, tx := range blk.GetTransactions() {
-		if i > 9999 {
-			return nil, nil, fmt.Errorf("unexpected number of transactions in block expected at most 9999 but got: %v, tx: %x", i, tx.GetHash())
+		if i >= TX_PER_BLOCK_LIMIT {
+			return nil, nil, fmt.Errorf("unexpected number of transactions in block expected at most %d but got: %v, tx: %x", TX_PER_BLOCK_LIMIT-1, i, tx.GetHash())
 		}
-		iReversed := reversePaddedIndex(i, 10000)
+		iReversed := reversePaddedIndex(i, TX_PER_BLOCK_LIMIT)
 
-		for j, idx := range tx.GetItx() {
-			if j > 999999 {
-				return nil, nil, fmt.Errorf("unexpected number of internal transactions in block expected at most 999999 but got: %v, tx: %x", j, tx.GetHash())
+		for j, itx := range tx.GetItx() {
+			if j >= ITX_PER_TX_LIMIT {
```

Comment:
Yes, I'm pretty sure `999999` was a typo, otherwise [reversePaddedIndex(j, 100000)](https://github.com/gobitfly/eth2-beaconchain-explorer/pull/2618/files/a076907a039c3a3e9073a222a4a2128f0833262c#diff-143a07e5aa31442d87e5dd9584d1525fc98e8b2fef161a08af276d65f60184acL1155) wouldn't really make sense.
Using the constants already paying off :smile: